<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>嘿嘿嘿张宏</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qss.qrc">
    <normaloff>:/images/IMG_8598.JPG</normaloff>:/images/IMG_8598.JPG</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="initiateButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
     </font>
    </property>
    <property name="text">
     <string>生成会话</string>
    </property>
   </widget>
   <widget class="QLabel" name="privateKeyLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>私有密钥：</string>
    </property>
   </widget>
   <widget class="QLabel" name="nNumber">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大素数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pNumber">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>原    根：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
     </font>
    </property>
    <property name="text">
     <string>未发现会话</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>协商密钥：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID1Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发起人：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID2Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>成员一：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ID3Label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>成员二：</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接数量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户ID:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192.168.43.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MicrosoftYaHei-Tahoma</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>广播地址：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="qss.qrc"/>
 </resources>
 <connections/>
</ui>
